frontend/src/components/organisms/ManageAttendees.tsx

```diff
@@ -125,6 +134,212 @@ const AttendeesTable = ({
+        <div className="mb-12">
+          <div className="text-center">
+            Create a new event with the same information as this one.
+          </div>
+          <div className="text-center">
+            Everything except the volunteer list will be copied over.
+          </div>
+        </div>
+        <div className="font-bold">Date and Time for New Event</div>
+        <div className="sm:space-x-0 grid grid-cols-1 sm:grid-cols-2 gap-4">
+          <div className="pb-0 sm:pb-0">
+            <Controller
+              name="startDate"
+              control={control}
+              rules={{ required: true }}
+              defaultValue={undefined}
+              render={({ field }) => (
+                <DatePicker
+                  label={<span className="font-medium">Start Date</span>}
+                  error={errors.startDate ? "Required" : undefined}
+                  {...field}
+                />
+              )}
+            />
+          </div>
+          <div className="pb-0 sm:pb-0">
+            <Controller
+              name="startTime"
+              control={control}
+              rules={{ required: true }}
+              defaultValue={undefined}
+              render={({ field }) => (
+                <TimePicker
+                  error={errors.startTime ? "Required" : undefined}
+                  label={<span className="font-medium">Start Time</span>}
+                  {...field}
+                />
+              )}
+            />
+          </div>
+        </div>
+        <div className="sm:space-x-0 grid grid-cols-1 sm:grid-cols-2 gap-4">
+          <div className="pb-0 sm:pb-0">
+            <Controller
+              name="endDate"
+              control={control}
+              rules={{ required: true }}
+              defaultValue={undefined}
+              render={({ field }) => (
+                <DatePicker
+                  error={errors.endDate ? "Required" : undefined}
+                  label={<span className="font-medium">End Date</span>}
+                  {...field}
+                />
+              )}
+            />
+          </div>
+          <div className="pb-0 sm:pb-0">
+            <Controller
+              name="endTime"
+              control={control}
+              rules={{ required: true }}
+              defaultValue={undefined}
+              render={({ field }) => (
+                <TimePicker
+                  error={errors.endTime ? "Required" : undefined}
+                  label={<span className="font-medium">End Time</span>}
+                  {...field}
+                />
+              )}
+            />
+          </div>
+        </div>
+        <Grid container spacing={2}>
+          <Grid item md={6} xs={12}>
+            <Button variety="secondary" onClick={handleClose}>
+              Cancel
+            </Button>
+          </Grid>
+          <Grid item md={6} xs={12}>
+            <Button type="submit">Duplicate</Button>
```

Comment:
use the `isPending` state available in the mutation for duplicating event for the loading prop for the button